test/Benchmarks/matmul_kernel_12x6x9.mlir

```diff
@@ -0,0 +1,27 @@
+// RUN: tpp-opt %s -map-linalg-to-tpp \
+// RUN:            -pre-bufferization \
```

Comment:
Can we drop `-pre-bufferization` in all tests? I need to remove this pass but I am waiting for @KavithaTipturMadhu to finish #129. I basically makes the relu and in-place operation for better bufferization. All the rest is now available upstream.

Reply:
Copy and paste from the benchmarks, which are old. Changing.